templates/vue/src/components/TapestryMain/index.vue

```diff
@@ -218,9 +217,6 @@ export default {
   },
   methods: {
     ...mapMutations(["select", "unselect", "clearSelection"]),
-    addRootNode() {
-      this.$root.$emit("add-node", null)
-    },
```

Comment:
I now see what this code is doing... it is apparently publishing the root node as the default node template at the same time as the node modal is opened. I removed it because I thought it was a bit redundant since we already open the node modal for the author to create the first node, and the handleSubmit will be called once the author clicks on the "Publish" button. But there may be a reason why we are immediately submitting the root node which I'm not aware of.

Reply:
Thanks for explaining. That makes sense. I created root nodes many times on this PR and haven't noticed anything incorrect, so deleting it seems reasonable to me as well.